joomla/joomla-cms
# [4.4] Use HelperFactory to get Helper for mod_quickicon

Pull Request for Issue # .

### Summary of Changes
This PR makes small changes to mod_quickicon code to use HelperFactory  to get it's Helper class. It has two benefits:

1. Consistent code with how we are getting Helper for other modules. For example https://github.com/joomla/joomla-cms/blob/4.2-dev/modules/mod_articles_latest/src/Dispatcher/Dispatcher.php#L41
2. Save the system from having to go to `bootModule` routine again before it can get module helper class.

### Testing Instructions
1. Use Joomla 4.3 nightly build
2. Apply patch
3. Access to administrator area of your site, look at Home Dashboard, Notifications section. Make sure the icons there are still being displayed (see the attached screenshot)

![quick_icons](https://user-images.githubusercontent.com/977664/222942694-4be6d0a8-7cc5-49c5-89eb-e8488c4dc80c.png)


### Actual result BEFORE applying this Pull Request
Works


### Expected result AFTER applying this Pull Request
Works


### Link to documentations
Please select:
